ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh

```diff
@@ -47,21 +47,86 @@ jumbo_mtu: $ENABLE_JUMBO_MTU
 install_rh_crucible: $CRUCIBLE
 rh_crucible_url: "$CRUCIBLE_URL"
 EOF
-
 envsubst < /tmp/all.yml > /tmp/all-updated.yml
 
 # Clean up previous attempts
 cat > /tmp/clean-resources.sh << 'EOF'
+echo 'Running clean-resources.sh'
+dnf install -y podman
 podman pod stop $(podman pod ps -q) || echo 'No podman pods to stop'
 podman pod rm $(podman pod ps -q)   || echo 'No podman pods to delete'
 podman stop $(podman ps -aq)        || echo 'No podman containers to stop'
 podman rm $(podman ps -aq)          || echo 'No podman containers to delete'
 rm -rf /opt/*
 EOF
 
-jetlag_repo=/tmp/jetlag-${LAB}-${LAB_CLOUD}-$(date +%s)
+# Pre-reqs
+cat > /tmp/prereqs.sh << 'EOF'
+echo "Running prereqs.sh"
+USER=$(curl -sS $QUADS_INSTANCE/cloud/$LAB_CLOUD\_ocpinventory.json | jq -r ".nodes[0].pm_user")
+PWD=$(curl -sS $QUADS_INSTANCE/cloud/$LAB_CLOUD\_ocpinventory.json | jq -r ".nodes[0].pm_password")
+if [[ "$TYPE" == "mno" ]]; then
+  HOSTS=$(curl -sS $QUADS_INSTANCE/cloud/$LAB_CLOUD\_ocpinventory.json | jq -r ".nodes[1:4+"$NUM_WORKER_NODES"][].pm_addr")
+elif [[ "$TYPE" == "sno" ]]; then
+  HOSTS=$(curl -sS $QUADS_INSTANCE/cloud/$LAB_CLOUD\_ocpinventory.json | jq -r ".nodes[1:1+"$NUM_SNO_NODES"][].pm_addr")
+fi
+echo "Hosts to be prepared: $HOSTS"
+# IDRAC reset
+if [[ "$PRE_RESET_IDRAC" == "true" ]]; then
+  echo "Resetting IDRACs ..."
+  for i in $HOSTS; do
+    echo "Resetting IDRAC of server $i ..."
+    podman run quay.io/quads/badfish:latest -v -H $i -u $USER -p $PWD --racreset
+  done
+  sleep 300
```

Comment:
I wonder if there's another way to check the iDRAC is up & running rather than an arbitrary sleep